### ISSUE TITLE
Transient failure on WorkerTaskMonitorTest

### ISSUE THREAD

**Issue by guobingkun** [druid-io/druid#2156]
https://travis-ci.org/druid-io/druid/jobs/98404476

```
io.druid.indexing.worker.WorkerTaskMonitorTest
testRunTask(io.druid.indexing.worker.WorkerTaskMonitorTest)  Time elapsed: 12.166 sec  <<< FAILURE!
java.lang.AssertionError: null
    at org.junit.Assert.fail(Assert.java:86)
    at org.junit.Assert.assertTrue(Assert.java:41)
    at org.junit.Assert.assertTrue(Assert.java:52)
    at io.druid.indexing.worker.WorkerTaskMonitorTest.testRunTask(WorkerTaskMonitorTest.java:200)
```

**Comment by guobingkun** [comment]
Tracked by #2154

**Comment by fjy** [comment]
Fixed by #2154
